### PR TITLE
BREAKING CHANGE: add support for boolean and number values in addLabels

### PR DESCRIPTION
**Pull request by jahtalab**
Part of #272

### CODE REVIEW

File: packages/rum-core/src/common/utils.js [hunk 1]
```diff
@@ -143,8 +143,13 @@ function isPlatformSupported() {
  */
 function setLabel(key, value, obj) {
   if (!obj || !key) return
-  var skey = removeInvalidChars(key)
-  if (value) {
+  const skey = removeInvalidChars(key)
+  let valueType = typeof value
+  if (
```
Comment (vigneshshanmugam) [inline packages/rum-core/src/common/utils.js]: This breaks the previous assumption when the value is `null`, We need to add a test when value is null.

File: packages/rum-core/src/common/utils.js [hunk 2]
```diff
@@ -134,18 +134,16 @@ function isPlatformSupported() {
...
+ * Support for boolean and number in the APM server landed in 6.7
+ * therefore, we keep these values unchange but convert any other
+ * type to string.
  */
 function setLabel(key, value, obj) {
   if (!obj || !key) return
   const skey = removeInvalidChars(key)
   let valueType = typeof value
   if (
+    value !== null &&
```
Comment (vigneshshanmugam) [inline packages/rum-core/src/common/utils.js]: we can merge these two lines to `value !== null && valueType !== 'undefined'` to `value != undefined` which handles both null and undefined.
